pagerduty.py

```diff
@@ -139,10 +140,10 @@ def _queued_files(self):
         # We need to sort the files by the timestamp.
         # This function extracts the timestamp out of the file name
         def file_timestamp(file_name):
-            return int(re.search('pd_(\d+)_', file_name).group(1))
+            return int(re.search('pd_(\d+)', file_name).group(1))
 
-        sorted_file_names = sorted(pd_file_names, key=file_timestamp)
-        return pd_file_names
+        sorted_file_names = sorted(pd_file_names, key=file_timestamp, reverse=True)
```

Comment:
If you enqueue a few things & then either run this code by hand or run it with print/log statements, we should be able to see how the sorting goes. :)